Sources/Plasma/Apps/plClient/CMakeLists.txt

```
@@ -169,6 +254,20 @@ if(PLASMA_EXTERNAL_RELEASE)
     set_target_properties(plClient PROPERTIES OUTPUT_NAME "UruExplorer")
 endif(PLASMA_EXTERNAL_RELEASE)
 
+if(APPLE)
+    get_target_property(OUTPUT_NAME plClient OUTPUT_NAME)
+    if(OUTPUT_NAME STREQUAL "OUTPUT_NAME-NOTFOUND")
+        get_target_property(OUTPUT_NAME plClient NAME)
+    endif()
+    install(
+        CODE "include(\"BundleUtilities\")
+            fixup_bundle(\"${CMAKE_INSTALL_PREFIX}/client/${OUTPUT_NAME}.app\" \"\" \"${searchDirs}\" IGNORE_ITEM Python)
+            execute_process(COMMAND \"codesign\" \"--remove-signature\" \"--deep\" \"-f\" \"${CMAKE_INSTALL_PREFIX}/client/${OUTPUT_NAME}.app\")
+            "
+        DESTINATION client
+    )
```

```suggestion
    install(
        CODE "include(\"BundleUtilities\")
            fixup_bundle(\"${CMAKE_INSTALL_PREFIX}/client/$<TARGET_PROPERTY:plClient,OUTPUT_NAME>.app\" \"\" \"${searchDirs}\" IGNORE_ITEM Python)
            execute_process(COMMAND \"codesign\" \"--remove-signature\" \"--deep\" \"-f\" \"${CMAKE_INSTALL_PREFIX}/client/$<TARGET_PROPERTY:plClient,OUTPUT_NAME>.app\")
            "
        DESTINATION client
    )
```

The target property `OUTPUT_NAME` is only available at generation time.

---

This change is not working. I'm investigating why.

---

Maybe instead of `$<TARGET_PROPERTY:plClient,OUTPUT_NAME>.app`, you could try `$<TARGET_FILE_NAME>`